lib/cfg.js

```diff
@@ -13,7 +13,13 @@ function resolvePath(unresolvedPath) {
 module.exports = function(main, opts) {
   var dir = main ? path.dirname(main) : '.'
   var c = read(dir)
-
+  if(c==null && dir!='.'){
+    dir = path.dirname('.');
+    c = read(dir);
+  }
+  if(c==null){
+    c={}
+  }
```

Comment:
Could you add a test case for this scenario? I am not very familiar with how Gulp works. Additionally, please update the docs on how the `.node-dev.json` lookup process works with this change: https://github.com/fgnass/node-dev#settings

Thanks :+1: